packages/rest/src/lib/CDN.ts

```diff
@@ -14,22 +14,22 @@ export class CDN {
 
 	/**
 	 * Generates an app asset URL for a client's asset.
-	 * @param clientID The client ID that has the asset
+	 * @param clientId The client id that has the asset
```

Comment:
The comment here shouldn't be updated, just the parameter name should be changed.

Reply:
`id` is correct, not ID. It's short for identifier, lowercase, not an acronym of Identity Document.

Reply:
Seems like it's still supposed to be uppercased:
![dictionary definition](https://user-images.githubusercontent.com/7608555/128616370-7daca8f2-0a2b-476d-b0e7-2defb245d184.png) https://www.merriam-webster.com/dictionary/id